include/glow/Base/Image.h

```diff
@@ -55,8 +55,16 @@ extern ImageChannelOrder imageChannelOrder;
 /// -image-layout flag.
 extern ImageLayout imageLayout;
 
-/// -use-imagenet-normalization flag.
-extern bool useImagenetNormalization;
+/// These are standard normalization factors for imagenet, adjusted for
+/// normalizing values in the 0to255 range instead of 0to1, as seen at:
+/// https://github.com/pytorch/examples/blob/master/imagenet/main.py
+static const float imagenetNormMean[] = {0.485 * 255.0, 0.456 * 255.0,
+                                         0.406 * 255.0};
+static const float imagenetNormStd[] = {0.229, 0.224, 0.225};
+
+/// default values for mean and stddev
+static const float zeroMean[] = {0.f, 0.f, 0.f};
```

Comment:
Why not `static const std::vector<float> zeroMean(max_tensor_dimensions, 0.f);` -- what if someone e.g. extends this with including the alpha channel?